dask/dataframe/core.py

```diff
@@ -1504,6 +1504,11 @@ def quantile(self, q=0.5, axis=0):
             Iterable of numbers ranging from 0 to 1 for the desired quantiles
         axis : {0, 1, 'index', 'columns'} (default 0)
             0 or 'index' for row-wise, 1 or 'columns' for column-wise
+
+        Note: this implementation will use t-digest for columns with floating
+              dtype if axis is set to 0 and `method` is set to `tdigest`.
+              Otherwise it falls back to the internal implementation.
```

Comment:
Doesn't it use tdigest only if that's specified explicitly?

Also, should put this as a parameter docstring:

```python
"""
...
method : {'default', 'tdigest', 'dask'}:
    What method to use. By default will use dask's internal custom algorithm (``'dask'``).
    If set to ``'tdigest'`` will use tdigest for floats and ints and fallback to the ``'dask'``
    otherwise.
...
"""
```

Reply:
Yes it does, your version is much better.